wiki/Music_theory/Syncopation/de.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+---
+
+# Synkope
+
+Eine **Synkope** ist die regelmäßige Akzentuierung oder Betonung von Noten an ungewöhnlichen Stellen. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```

Comment:
Should we distinguish `(eine) Synkope` and `Synkope/Synkopieren`? According to the German sources, `eine Synkope`, which is what you're using here, only describes one instance of this irregular rhythm construct. The english osu-wiki article refers to `Syncopation` (not `a syncopation` (singular)), which means the process of using syncopations regularly throughout the song. To be technically correct here, I think we'd need to use `Synkopieren` (a `substantiviertes Verb`), which doesn't sound great though. Something like this maybe:

```suggestion
**Synkopieren** beschreibt die regelmäßige rhythmische Betonung von Noten, die üblicherweise nicht betont werden. Die [Taktart](/wiki/Music_theory/Time_signature) eines Musikstücks gibt einen gleichmäßigen [Rhythmus](/wiki/Music_theory/Rhythm) aus starken und schwachen Beats vor, wohingegen ein synkopischer Rhythmus dieser Erwartung widerspricht, indem die Akzente in den Offbeat oder auf unbetonte Stellen versetzt werden. Beim [Beatmapping](/wiki/Beatmapping) sind Synkopen normalerweise zu erkennen, wenn [Hit-Objekte](/wiki/Gameplay/Hit_object) regelmäßig zeitlich verschoben werden, wie zum Beispiel auf roten oder blauen Strichen auf der Zeitleiste.
```